template/template.go

```diff
@@ -69,36 +80,52 @@ func FromGlobs(paths []string, options ...Option) (*Template, error) {
 			return nil, err
 		}
 		defer f.Close()
```

Comment:
Technically not part of your change but I would appreciate it if now that we're here, we can take the opportunity to fix this potential resource leak.

This defer is called within the for loop so closing the file is not actually called until the end of the loop.

I say we aim for the following:

```go
	for _, file := range defaultTemplates {
		f, err := asset.Assets.Open(path.Join("/templates", file))
		if err != nil {
			return nil, err
		}
		
		if err := t.Parse(f); err != nil {
		        f.Close()
			return nil, err
		}
		f.Close()
	}
```

Reply:
Given this has been waiting for a while - I pushed two comments to address my own comments.

Thank you very much for your contribution!